config/development.json

```diff
@@ -12,6 +12,6 @@
     "webSocketPort": 9222
   },
   "firefox": {
-    "webSocketPort": 9000
+    "webSocketPort": 6080
```

Comment:
Are you sure? I see it used [here](https://github.com/jlongster/debugger.html/blob/master/public/js/clients/firefox.js#L52). How else would the debugger client know to which port it should connect?